test/parameterizer/claimRewards.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-env mocha */
+/* global artifacts assert contract */
+const Parameterizer = artifacts.require('./Parameterizer.sol');
+const Token = artifacts.require('EIP20.sol');
+
+const fs = require('fs');
+const BN = require('bn.js');
+const utils = require('../utils');
+
+const config = JSON.parse(fs.readFileSync('./conf/config.json'));
+const paramConfig = config.paramDefaults;
+
+const bigTen = number => new BN(number.toString(10), 10);
+
+contract('Parameterizer', (accounts) => {
+  describe('Function: claimRewards', () => {
+    const [proposer, challenger, voterAlice] = accounts;
+
+    it('should give the correct number of tokens to a voter on the winning side.', async () => {
+      const parameterizer = await Parameterizer.deployed();
+      const token = Token.at(await parameterizer.token.call());
+      const voting = await utils.getVoting();
+
+      const voterAliceStartingBalance = await token.balanceOf.call(voterAlice);
+
+      // propose reparam
+      const proposalReceipt = await utils.as(proposer, parameterizer.proposeReparameterization, 'voteQuorum', '51');
+      const { propID } = proposalReceipt.logs[0].args;
+
+      // challenge reparam
+      const challengeReceipt =
+        await utils.as(challenger, parameterizer.challengeReparameterization, propID);
+      const { challengeID } = challengeReceipt.logs[0].args;
+
+      // commit vote
+      await utils.commitVote(challengeID, '1', '10', '420', voterAlice);
+      await utils.increaseTime(paramConfig.pCommitStageLength + 1);
+
+      // reveal vote
+      await utils.as(voterAlice, voting.revealVote, challengeID, '1', '420');
+      await utils.increaseTime(paramConfig.pRevealStageLength + 1);
+
+      // process reparam
+      await parameterizer.processProposal(propID);
+
+      // array args
+      const challengeIDs = [challengeID];
+      const salts = ['420'];
+
+      // multi claimRewards, arrays as inputs
+      await utils.as(voterAlice, parameterizer.claimRewards, challengeIDs, salts);
+      await utils.as(voterAlice, voting.withdrawVotingRights, '10');
+
+      // state assertion
+      const voterAliceFinalBalance = await token.balanceOf.call(voterAlice);
+      // expected = starting balance + voterReward
+      const voterAliceExpected = voterAliceStartingBalance.add(utils.multiplyByPercentage(
```

Comment:
Lets replace this complicated math with the parameterizer's `voterReward` function?